docs/sql-reference/functions-reference/semi-structured-functions/array-functions.md

```diff
@@ -771,7 +774,7 @@ INDEX_OF(<arr>, <x>)
 
 ```sql
 SELECT
-    INDEX_OF([1, 3, 5, 7], 5) AS res;
+	INDEX_OF([ 1, 3, 5, 7 ], 5) AS res;
```

Comment:
I'm surprised by this spacing requirement. Is this a standard thing, or a quirk of ours (that is, a bug)?

Reply:
It looks like https://bitbucket.org/packdb/packui/src/28ddc0f8d1c0d59a5e43c3fa6ed523ec0d6873d8/src/components/QueryEditor/hooks/useFormatSQL.ts#lines-6 is the config for our format command and I think https://www.npmjs.com/package/sql-formatter is the library. It has a demo https://zeroturnaround.github.io/sql-formatter/. I did try this out on the demo and it does add spacing around the array